prime-router/metadata/file_name_templates/file-name-templates.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: standard
+  lowerCase: true
+  elements:
+    - schemaBaseName()
```

Comment:
FWIW, this is off your main topic, but it would make the files more readable if our default name had the receiver_org name instead of the schema name.  I feel like it used to be that way, but changed at some point?